.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for more information:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+# https://containers.dev/guide/dependabot
+
+version: 2
+updates:
+ - package-ecosystem: "devcontainers"
+   directory: "/"
+   schedule:
+     interval: weekly
```

Comment:
Question: Any idea what dependabot checks for? Mostly curious

Reply:
For the dependencies in the devcontainer like `"ghcr.io/devcontainers/features/docker-in-docker:2"`, it will upgrade if that becomes available. Here are more docs: https://containers.dev/guide/dependabot